src/Multics.xml

```diff
@@ -27,6 +27,12 @@
          successor in interest by change in name only to Honeywell Bull Inc. and Honeywell Information Systems
          Inc.</p>
       </optional>
+	  <optional>
+	  .
```

Comment:
Looking at [the OSI text][1], I read this extra period as part of the previous paragraph.  That would make this:

```xml
<p>This edition… Inc.<optional> .</optional></p>
```

[1]: https://opensource.org/licenses/Multics

Reply:
This would create an optional text inside another optional text, which is not currently supported by the tools.

Reply:
> This would create an optional text inside another optional text, which is not currently supported by the tools.

Ah.  I''m fine leaving this alone until #462 is settled then.